test/parallel/test-stream-finished.js

```diff
@@ -104,7 +104,9 @@ const { promisify } = require('util');
   }));
 
   rs.push(null);
-  rs.emit('close'); // Should not trigger an error
```

Comment:
can you add another test where this is happening? What are we expecting?

Reply:
added another test